src/system.hpp

```diff
@@ -9,8 +9,8 @@
  * @file system.hpp
  * @brief Contains functions that interact with the system or runtime environment.
  */
-
-#pragma once
```

Comment:
All of LvArray is using `#pragma once`, did you not have any issues with it before?